scripts/vim-patch.sh

```
@@ -256,7 +256,8 @@ list_vim_patches() {
   for vim_commit in ${vim_commits}; do
     local is_missing
     local vim_tag
-    vim_tag="$(cd "${VIM_SOURCE_DIR}" && git describe --tags --exact-match "${vim_commit}" 2>/dev/null)"
+    # This fails for untagged commits (e.g., runtime file updates) so mask the return status
+    vim_tag="$(cd "${VIM_SOURCE_DIR}" && git describe --tags --exact-match "${vim_commit}" 2>/dev/null)" || true
```

Can it instead be moved back to line 258 so the subshell swallows the error?